renderdoc/os/win32/win32_shellext.cpp

```diff
@@ -449,6 +453,7 @@ struct RDCThumbnailProvider : public IThumbnailProvider, IInitializeWithStream
       decompressed.resize(thumbheight * thumbwidth * 4);
       unsigned char decompressedBlock[decompressedBlockMaxSize];
       unsigned char greenBlock[16];
+      unsigned short decompressedBC6[48];
```

Comment:
I'd prefer an explicitly sized format `uint16_t` here.

Reply:
Changed to `uint16_t`.